api/src/pcapi/routes/backoffice/filters.py

```diff
@@ -267,7 +267,17 @@ def format_booking_cancellation(
             bookings_models.BookingCancellationReasons.FRAUD
             | educational_models.CollectiveBookingCancellationReasons.FRAUD
         ):
-            return "Fraude"
+            return "Fraude avérée"
```

Comment:
Je vois un effet de bord : on verra « Fraude avérée » pour toute réservation annulée par la fraude avant l'arrivée des nouvelles raisons que tu ajoutes ici, y compris celles liées à l'inéligibilité ou simple suspicion. Cela peut fausser la compréhension. Est-ce acceptable ? Ou serait-il préférable d'ajouter une raison spécifique pour fraude avérée, tout en conservant `FRAUD` comme raison imprécise ?

Reply:
J'ai demandé à JB, et il m'a répondu "Pas grave"